src/app/header/header.component.scss

```diff
@@ -0,0 +1,76 @@
+.container {
+  border-bottom: 1px solid rgb(245, 234, 234);
+  box-shadow: 0px 5px 5px rgb(241, 238, 238); /*  x axis , y axis , blur , color */
+}
+
+.col-4 {
```

Comment:
Please see if we can re-factor this file a bit more. Looks like we're writing a lot of CSS for seemingly simple tasks.

Reply:
done